examples/discovery-search-app/src/App.js

```diff
@@ -85,9 +88,20 @@ function SearchPage() {
   );
 }
 
-function PreviewPage() {
+function PreviewPage(props) {
```

Comment:
-> `({ document })`, therefore you won't need to do `props.document` several times below.